rskj-core/src/test/java/co/rsk/peg/BridgeSerializationUtilsTest.java

```diff
@@ -87,6 +87,94 @@ class BridgeSerializationUtilsTest {
     private static final Address ADDRESS = BitcoinTestUtils.createP2PKHAddress(NETWORK_PARAMETERS, "first");
     private static final Address OTHER_ADDRESS = BitcoinTestUtils.createP2PKHAddress(NETWORK_PARAMETERS, "second");
 
+    @Test
+    void serializeAndDeserializeBtcTransaction_withValidDataAndInputs_shouldReturnEqualResults() {
+        // Arrange
+        BtcTransaction fundTx = new BtcTransaction(NETWORK_PARAMETERS);
```

Comment:
`fundTx` might give the impression this is something related to svp, what about we call it `prevTx` here as well

Reply:
agree! Thanks